conf/machine/hikey.conf

```diff
@@ -17,6 +17,11 @@ MACHINE_FEATURES = "usbhost usbgadget alsa screen wifi bluetooth efi"
 
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-hikey"
 
+PREFERRED_PROVIDER_virtual/libgles1 ?= "mali450-userland-drm"
+PREFERRED_PROVIDER_virtual/libgles2 ?= "mali450-userland-drm"
+PREFERRED_PROVIDER_virtual/egl ?= "mali450-userland-drm"
```

Comment:
It requires a binary blob not available for distribution yet. See:
https://github.com/kuscsik/meta-los/blob/hikey/recipes-graphics/mali-userland/mali450-userland-drm_binary.bb